packages/components/src/avatar/src/Avatar.css

```diff
@@ -30,6 +30,7 @@
 /* IMAGE */
 .o-ui-avatar-image {
     width: 100%;
+    object-fit: cover;
```

Comment:
I would add a test to validate that a non square image is indeed acting like we want.